diff -ur x2divx-0.4.orig/Makefile x2divx-0.4/Makefile
--- x2divx-0.4.orig/Makefile	Mon Oct 23 23:42:50 2000
+++ x2divx-0.4/Makefile	Mon Oct 30 12:09:17 2000
@@ -1,10 +1,12 @@
-all: avi2divx mpeg2divx
+all: libs avi2divx mpeg2divx 
 
+libs:
+	(cd libmpeg3-1.2.1 && ./configure && make && cd ..)                     
 avi2divx:
-	g++ -I/usr/local/include/avifile -o avi2divx avi2divx.cpp -laviplay -lpthread
+	g++ $(OPT) -I/usr/X11R6/include/avifile -L/usr/X11R6/lib -o avi2divx avi2divx.cpp -laviplay -lpthread
 mpeg2divx:
-	g++ -I./libmpeg3-1.2.1 -I/usr/local/include/avifile -o mpeg2divx mpeg2divx.cpp -laviplay -lpthread libmpeg3-1.2.1/libmpeg3.a
+	g++ $(OPT) -I./libmpeg3-1.2.1 -I/usr/X11R6/include/avifile -L/usr/X11R6/lib -o mpeg2divx mpeg2divx.cpp -laviplay -lpthread libmpeg3-1.2.1/libmpeg3.a
 install:
-	install -m 755 avi2divx mpeg2divx /usr/local/bin
+	install -m 755 avi2divx mpeg2divx $(DESTDIR)$(bindir)
 clean:
 	rm -f avi2divx mpeg2divx	
Only in x2divx-0.4: Makefile~
